rero/rero-ils
# data model: language, identifiedBy

* transform Marc21 fields containing identifiers
* transform Marc21 fields containing language codes 
* update for 10k and 300 fixture files 
* json schema and mapping and unit testing for identifiedBy
* json schema and mapping and unit testing for language
* adapt document views and facets
* adapt editor
* regenerates the item json files for fixtures
* adapts ebook

Co-Authored-by: Gianni Pante <gianni.pante@rero.ch>
Co-Authored-by: Peter Weber <peter.weber@rero.ch>
Co-Authored-by: Aly Badr aly.badr@rero.ch
Co-Authored-by: Bertrand Zuchuat bertrand.zuchuat@rero.ch
Co-Authorred-by: Johnny Mariéthoz Johnny.Mariethoz@rero.ch

# To test:
* bootstrap, setup
* make sure the two fields are correctly displayed and editable in the interface
